sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
Sorry - yes it was more than one line - I forgot about the first two bullets you listed.  Only the last one was "real" to me so I blocked out the other 2